File: SRC - PERL/00 - Estruturas Sequenciais/05.pl
#!/usr/bin/env perl
use v5.10;

use strict;
use warnings;

use utf8;
use open qw(:std :utf8);

# Faça um programa que receba o salário base de um funcionário, calcule e mostre o salário a receber, sabendo-se que o funcionário tem gratificação de 5% sobre o salário base e paga imposto de 7% sobre este salário'

print "Digite o salário base > ";
my $salario_base = <>;
chomp $salario_base;

my $gratificacao = $salario_base * 5 / 100;
my $imposto = $salario_base * 7 / 100;

my $novo_salario = $salario_base + $gratificacao - $imposto;

printf "O novo salário é R\$ %.2f", $novo_salario;
say "";

File: startingPoint/02-EstruturaSequencial/Exercicios_Propostos/05.py
#!/usr/bin/python
# coding: latin-1

# 05 - Faça um programa que receba o salário base de um funcionário, calcule e mostre o salário a receber, sabendo-se que o funcionário
# tem gratificação de 5% sobre o salário base e paga imposto de 7% sobre este salário.

sal_base = float(input("Digite o valor do seu salário aqui: "))

grat = sal_base * 5 / 100
imp = sal_base * 7 / 100

sal_receber = (sal_base + grat - imp)

print("Seu salário será: R$", sal_receber)
